.changeset/strange-olives-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
+---
+
+Migrated the package from using moment to luxon. #4278
```

Comment:
For vale complaining about `luxon`
```suggestion
Migrated the package from using moment to Luxon. #4278
```

FYI the vale vocab is managed in `.github/styles/vocab.txt`.